pkg/gohai/cpu/cpu_windows.go

```diff
@@ -59,96 +46,110 @@ type SYSTEM_INFO struct {
 	wProcessorRevision      uint16
 }
 
-// CPU_INFO contains information about cpu, eg. number of cores, cache size
-//
-//nolint:revive
-type CPU_INFO struct {
-	numaNodeCount int    // number of NUMA nodes
-	pkgcount      int    // number of packages (physical CPUS)
-	corecount     int    // total number of cores
-	logicalcount  int    // number of logical CPUS
-	l1CacheSize   uint32 // layer 1 cache size
-	l2CacheSize   uint32 // layer 2 cache size
-	l3CacheSize   uint32 // layer 3 cache size
-	//nolint:unused
-	relationGroups int // number of cpu relation groups
-	//nolint:unused
-	maxProcsInGroups int // max number of processors
-	//nolint:unused
-	activeProcsInGroups int // active processors
+// cpuInfo holds CPU information
+type cpuInfo struct {
+	corecount           int
+	logicalcount        int
+	pkgcount            int
+	numaNodeCount       int
+	relationGroups      int
+	maxProcsInGroups    int
+	activeProcsInGroups int
+	l1CacheSize         uint64
+	l2CacheSize         uint64
+	l3CacheSize         uint64
+	VendorID            string
```

Comment:
Since these fields get copied over to the exported `Info` struct, how about lower-casing these next few fields to make them unexported? Switching visibility mid-declaration seems strange to me.